scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -410,7 +393,7 @@ export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapp
                                     getExportModal(getLatestItem, handleUnsavedChanges, hasUnsavedChanges),
                                     getTranslateModal(getLatestItem),
                                     ...authoringActionsFromExtensions,
-                                    ...articleActionsFromExtensions,
+                                    // ...articleActionsFromExtensions,
```

Comment:
this should probably not be commented out?

Reply:
Yes, this should be completely removes, since if we leave it, both `list item three dots menu` actions & `authoring three dots menu` actions show up in `authoring three dots menu`, thus causing duplication of actions. Regardless of duplication this shouldn't be there, since we agreed that getActions is only for the list item